test/main.js

```diff
@@ -163,237 +161,199 @@ describe('node-fetch', () => {
-			return res.text().then(result => {
-				expect(res.bodyUsed).to.be.true;
-				expect(result).to.be.a('string');
-				expect(result).to.equal('<html></html>');
-			});
-		});
+		const res = await fetch(url);
+		expect(res.headers.get('content-type')).to.equal('text/html');
+		const result = await res.text();
+		expect(res.bodyUsed).to.be.true;
+		expect(result).to.be.a('string');
+		expect(result).to.equal('<html></html>');
 	});
 
-	it('should accept json response', () => {
+	it('should accept json response', async () => {
 		const url = `${base}json`;
-		return fetch(url).then(res => {
-			expect(res.headers.get('content-type')).to.equal('application/json');
-			return res.json().then(result => {
-				expect(res.bodyUsed).to.be.true;
-				expect(result).to.be.an('object');
-				expect(result).to.deep.equal({name: 'value'});
-			});
-		});
+		const res = await fetch(url);
+		expect(res.headers.get('content-type')).to.equal('application/json');
+		const result = await res.json();
+		expect(res.bodyUsed).to.be.true;
+		expect(result).to.be.an('object');
+		expect(result).to.deep.equal({name: 'value'});
 	});
 
-	it('should send request with custom headers', () => {
+	it('should send request with custom headers', async () => {
 		const url = `${base}inspect`;
 		const options = {
 			headers: {'x-custom-header': 'abc'}
 		};
-		return fetch(url, options).then(res => {
-			return res.json();
-		}).then(res => {
-			expect(res.headers['x-custom-header']).to.equal('abc');
-		});
+		const res = await fetch(url, options);
+		const result = await res.json();
+		expect(result.headers['x-custom-header']).to.equal('abc');
 	});
 
-	it('should accept headers instance', () => {
+	it('should accept headers instance', async () => {
 		const url = `${base}inspect`;
 		const options = {
 			headers: new Headers({'x-custom-header': 'abc'})
 		};
-		return fetch(url, options).then(res => {
-			return res.json();
-		}).then(res => {
-			expect(res.headers['x-custom-header']).to.equal('abc');
-		});
+		const res = await fetch(url, options);
+		const json = await res.json();
+		expect(json.headers['x-custom-header']).to.equal('abc');
 	});
 
-	it('should accept custom host header', () => {
+	it('should accept custom host header', async () => {
 		const url = `${base}inspect`;
 		const options = {
 			headers: {
 				host: 'example.com'
 			}
 		};
-		return fetch(url, options).then(res => {
-			return res.json();
-		}).then(res => {
-			expect(res.headers.host).to.equal('example.com');
-		});
+		const res = await fetch(url, options);
+		const json = await res.json();
+		expect(json.headers.host).to.equal('example.com');
 	});
 
-	it('should accept custom HoSt header', () => {
-		const url = `${base}inspect`;
-		const options = {
-			headers: {
-				HoSt: 'example.com'
-			}
-		};
-		return fetch(url, options).then(res => {
-			return res.json();
-		}).then(res => {
-			expect(res.headers.host).to.equal('example.com');
-		});
-	});
```

Comment:
Why was this removed?

Reply:
we have two identical tests: https://github.com/node-fetch/node-fetch/blob/1028f8370844060ec81279a9b956b0c22d611cf3/test/main.js#L226-L252

Reply:
One was setting the header as `hOsT`, and tested that it was normalised into `Host`

Reply:
Oh... hmm

Reply:
guess i can take it back